### PR TITLE
Finished Test Method for Robot

### PR DESCRIPTION
**Pull request by Dmac9244**
I finished the code for the Voltage/Distance test. Currently, it is set up so that whenever raw button 14 is pressed, the robot will move forward at i * .1 Voltage for 5 seconds, where it will increment i by one, reset the timer, and wait until button 14 is pressed again. 

I also commented out the thing that looked like it may have checked the joy stick for values and set them to the motors, and I may have accidentally deleted some lines that are necessary.

### CODE REVIEW

**Comment by Dmac9244** [comment]
Is it possible to add branches? As in, a branch for me, one for Brendan, and such, so we don't have to submit pull requests to the master branch?

**Comment by martinpirringer** [comment]
Ask Mr. Kiedes he is in charge of branches

And did you change stuff in testperiodic? we need that for testing - you
can add your code to that even though it might fit better into
autonomous for now.

If you can run the test tomorrow

**Comment by Dmac9244** [comment]
@martinpirringer 
The only thing I changed was that I commented out a line that seemed like it might mess with the voltage of the motors if the joystick is moved.

You should be able to see the changes I made by switching to the "Files Changed" tab under "Finished Test Method for Robot" at the top of the page.

File: src/org/usfirst/frc/team1989/robot/Robot.java [hunk 1]
```diff
@@ -38,7 +38,12 @@
 	Timer t1 = new Timer();
 	Servo s1 = new Servo(0);
 
-	// JsScaled utilityStick = new JsScaled(1);
```
Comment (Dmac9244) [inline src/org/usfirst/frc/team1989/robot/Robot.java]: I may have mistakenly deleted this.

**Comment by martinpirringer** [comment]
Will look at it after work


Martin

**Comment by mrkiedes** [comment]
Good Afternoon,

Will take a look at it today.  I will also look into and create branches
(although I think anyone can make a branch).

Sincerely,
Mr. Kiedes

**Pull request merged**